packages/story-editor/src/components/footer/pagepreview/index.js

```diff
@@ -67,20 +66,13 @@ const Page = styled.button`
     border-style: solid;
     border-width: 1px;
     border-radius: 8px;
-    border-color: ${({ isActive, theme }) =>
-      isActive ? theme.colors.border.defaultActive : 'transparent'};
+    border-color: ${({ isInteractive, isActive, theme }) =>
+      isInteractive
+        ? `${rgba(theme.colors.border.selection, isActive ? 1 : 0.7)}`
```

Comment:
Any way to skip using more than one ternary operators at a time here (to improve readability)?
I can see that previously, `isInteractive` was below, is this change related to the Safari issue?

Reply:
Will try and simplify but also not related,  was trying to comply with stylelint

Reply:
Alright, would it not pass without the change? Feel free to just revert it to what it was before not to spend too much time on it.